adapters/kobler/kobler.go

```diff
@@ -0,0 +1,146 @@
+package kobler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint    string
+	devEndpoint string
+}
+
+const (
+	DEV_BIDDER_ENDPOINT = "https://bid-service.dev.essrtb.com/bid/prebid_server_rtb_call"
+	SUPPORTED_CURRENCY  = "USD"
```

Comment:
Nitpick: Per Go naming standards, please use mixedCase instead of SCREAMING_CAPS. Consider renaming to `devBidderEndpoint` and `supportedCurrency`.